### PR TITLE
Implement: should display active items only

### PR DESCRIPTION
**Pull request by jabraok**

### CODE REVIEW

File: app/controllers/customers/show/location.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default Ember.Controller.extend({
   @computed("items", "model.itemDesires", "model.itemCreditRates")
```
Comment (brancusi) [inline app/controllers/customers/show/location.js]: We need the isProduct and active key

File: app/controllers/sales-orders/show.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ export default Ember.Controller.extend({
     return items.filter(item => {
       const matchingOrderItem = orderItems.any(oi => oi.get("item.id") === item.get("id"));
 
-      return item.get("isSold") && !matchingOrderItem
+      return item.get("isSold") && item.get("active") && !matchingOrderItem
```
Comment (brancusi) [inline app/controllers/sales-orders/show.js]: Need the active and isSold key in the computed above

File: app/controllers/purchase-orders/show.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ export default Ember.Controller.extend({
       const isTheSameCompany = item.get("company.id") === companyId;
       const matchingOrderItem = orderItems.any(oi => oi.get("item.id") === item.get("id"));
 
-      return isTheSameCompany && !matchingOrderItem
+      return item.get("active") && isTheSameCompany && !matchingOrderItem
```
Comment (brancusi) [inline app/controllers/purchase-orders/show.js]: need item.@each.{active}

Comment (jabraok) [inline app/controllers/purchase-orders/show.js]: Sorry about this inconvenience. I just updated the code and merge the latest code. Please check it again

File: app/controllers/purchase-orders.js [hunk 4]
```diff
@@ -1,34 +1,36 @@
...
+import computed from "ember-computed-decorators";
 
-const { filterBy } = Ember.computed;
 const tomorrow = moment().add(1, "days").format("YYYY-MM-DD");
 
 export default Ember.Controller.extend({
   deliveryDate: tomorrow,
 
-  @computed('orders.@each.{deliveryDate}', 'deliveryDate')
+  @computed("orders.@each.{deliveryDate}", "deliveryDate")
```
Comment (brancusi) [inline app/controllers/purchase-orders.js]: We need to add `isPurchaseOrder` as a key.

Basically anything that is used in the computed prop should be added up in the computed.

Comment (jabraok) [inline app/controllers/purchase-orders.js]: Thanks Aram. Please review one more time

File: app/controllers/purchase-orders.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ const tomorrow = moment().add(1, "days").format("YYYY-MM-DD");
 export default Ember.Controller.extend({
   deliveryDate: tomorrow,
 
-  @computed("orders.@each.{deliveryDate}", "deliveryDate")
+  @computed("orders.@each.{deliveryDate,isPurchaseOrder}", "deliveryDate")
   filteredOrders(orders, deliveryDate) {
     return orders.filter(order => {
       const matchesDate = order.get("deliveryDate") === deliveryDate;
```
Comment (brancusi) [inline app/controllers/purchase-orders.js]: Does the missing ; not cause a test fail?
`return matchesDate && order.get("isPurchaseOrder")` -> `return matchesDate && order.get("isPurchaseOrder");`

File: app/controllers/sales-orders.js [hunk 6]
```diff
@@ -7,7 +7,7 @@ const tomorrow = moment().add(1, "days").format("YYYY-MM-DD");
 export default Ember.Controller.extend({
   deliveryDate: tomorrow,
 
-  @computed("salesOrders.@each.{deliveryDate}", "deliveryDate")
+  @computed("salesOrders.@each.{deliveryDate,isSalesOrder}", "deliveryDate")
   filteredSalesOrders(salesOrders, deliveryDate) {
     return salesOrders.filter(order => {
       const matchesDate = order.get("deliveryDate") === deliveryDate;
```
Comment (brancusi) [inline app/controllers/sales-orders.js]: Same as above

File: app/controllers/purchase-orders.js (hunk 5 above)
Comment (jabraok) [inline app/controllers/purchase-orders.js]: All the tests are passed. I'll add the colon

Comment (jabraok) [inline app/controllers/purchase-orders.js]: I updated the code. However, it's kind of manually. Is there anyway we check it by ESLint?

**Pull request merged**